pyntc/devices/aireos_device.py

```diff
@@ -467,7 +486,7 @@ def close(self):
             self._connected = False
 
     def config(self, command, **netmiko_args):
-        """
+        r"""
```

Comment:
Seems like this passes all tests but there is an extra `r"""` in front of the docstring.

Reply:
Hey Dr.X, pydocstyle recommends putting an r before docstrings that contains backslashes. It is error D301 on this page.